### ISSUE TITLE
Unable to build solution using dev container

### ISSUE THREAD

**Issue by phoshobro** [pnp/sp-dev-fx-webparts#3433]
### Disclaimer

Yes

### Sample

https://github.com/pnp/sp-dev-fx-webparts/tree/main/samples/react-directory

### Author(s)

@joaojmendes @petkir @sudharsank @Abderahman88  

### What happened?

I am unable to build this solution even using a dev container.  I have set up a container for SPFX version 1.11 using https://hub.docker.com/r/m365pnp/spfx and appending the 1.11.0 version tag.  

This should be setting up the ideal dev environment for SPFX 1.11.0 and I can confirm the node version the container is using is node version 10.22 and yet I am unable to build the solution due to this error:

![image](https://user-images.githubusercontent.com/45148552/214606092-f0bc59c2-75e5-4334-bde5-abc5800ccb60.png)

After trying to find a solution on google, worker_threads were only introduced as of node version 10.5... I attempted to use node 10.5 instead of 10.22 and it still fails.

Any idea what is wrong?  I have absolutely no issues building any other solution, no matter what SPFX version they are using.  This seems to be the only one causing issues for me.

I have even attempted to setup my personal workstation to match all the correct versions, no issues as per spfx doctor and yet the same error occurs!

![Screenshot 2023-01-25 111545](https://user-images.githubusercontent.com/45148552/214616786-4dd64672-4301-4fb7-895e-dd4415cbdd43.jpg)

### Steps to reproduce

1. Once environment is set up as per SPFX version, run npm install
2. gulp build
3. error occurs for worker_threads

### Expected behavior

Project should build as required node version is used.

### Target SharePoint environment

SharePoint Online

### Developer environment

Windows

### Browsers

- [ ] Internet Explorer
- [ ] Microsoft Edge
- [ ] Google Chrome
- [ ] FireFox
- [ ] Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### What version of Node.js is currently installed on your workstation?

10.22

### What version of Node.js is required by the sample?

10

### Paste the results of SPFx doctor

![Screenshot 2023-01-25 111545](https://user-images.githubusercontent.com/45148552/214616792-503cfa59-c01d-46a7-94c2-499416646e6e.jpg)

**Comment by petkir** [comment]
Hi @phoshobro,

it looks like a version missmatch in package.json
```
"@microsoft/sp-property-pane": "1.16.1",
"@microsoft/sp-webpart-base": "1.16.1",
...
"@microsoft/sp-build-web": "1.16.1",
```

correct version should be "1.11.0"

**Comment by phoshobro** [comment]
> Hi @phoshobro,
> 
> it looks like a version missmatch in package.json
> 
> ```
> "@microsoft/sp-property-pane": "1.16.1",
> "@microsoft/sp-webpart-base": "1.16.1",
> ...
> "@microsoft/sp-build-web": "1.16.1",
> ```
> 
> correct version should be "1.11.0"

Hi @petkir thank you for the follow up! I am wondering, is this an issue on my end, something to do with my environment or is this an issue in the sample?

Also, now that I have fixed the package.json as suggested, I get a new error.  Note that I have deleted the node_modules and re-did an npm install.  

![image](https://user-images.githubusercontent.com/45148552/215530906-e0dc092a-fb1f-471f-92ee-9b11899edd53.png)

Thanks again for help!
